homeserver.yaml

```diff
@@ -0,0 +1,37 @@
+# Configuration file for Synapse.
+#
+# This is a YAML file: see [1] for a quick introduction. Note in particular
+# that *indentation is important*: all the elements of a list or dictionary
+# should have the same indentation.
+#
+# [1] https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html
+#
+# For more information on how to configure Synapse, including a complete accounting of
+# each option, go to docs/usage/configuration/config_documentation.md or
+# https://matrix-org.github.io/synapse/latest/usage/configuration/config_documentation.html
+server_name: "my.matrix.host"
+pid_file: /data/homeserver.pid
+listeners:
+  - port: 8008
+    tls: false
+    type: http
+    x_forwarded: true
+    resources:
+      - names: [client, federation]
+        compress: false
+database:
+  name: sqlite3
+  args:
+    database: /data/homeserver.db
+log_config: "/data/my.matrix.host.log.config"
+media_store_path: /data/media_store
+registration_shared_secret: "n5M1pehEj:fr+i.+4o1P0My7*~V,iKfQ8tawjy*_0RGuxXMNSu"
+report_stats: true
+macaroon_secret_key: "wboT^:0l6YL;0:XtDezWsWp5pY0=3QdBbhY_nsSV2Je0ffNQ;q"
+form_secret: "4rTUH6S3,E6AO2rcb36^s,mu64ROITAb3W8U0,@s2bA:NF2*6L"
+signing_key_path: "/data/my.matrix.host.signing.key"
+trusted_key_servers:
+  - server_name: "matrix.org"
+
+
+# vim:ft=yaml
```

Comment:
Final new line?

Reply:
Yes, though that file is an old artifact so I deleted it altogether, it's not needed. Thanks!